SmartDeviceLink/SDLFocusableItemLocator.h

```diff
@@ -16,6 +16,11 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (nonatomic, assign) BOOL enableHapticDataRequests;
 
+/**
+ The projection view controller associated with the Haptic Manager
+ */
+@property (nonatomic, weak) UIViewController *viewController;
```

Comment:
Was there a benefit to moving from UIWindow to UIViewController?

Reply:
No, but because `CarWindow` needs a `UIViewController`, it's easier to have developers set just the VC than the `UIWindow` and the VC both.